js/src/cli.js

```diff
@@ -590,7 +591,17 @@ function checkFiles(parsed) {
     // assume any remaining args are files
     argv.remain.forEach(function(f) {
       if (f !== '-') {
-        parsed.files.push(path.resolve(f));
+        var foundFiles = glob(f, {
+          sync: true,
+          absolute: true,
+          ignore: ['**/node_modules/**', '**/.git/**']
+        });
+
+        if (foundFiles && foundFiles.length) {
+          foundFiles.forEach(function(file) {
+            parsed.files.push(file);
```

Comment:
@StefanKoenen 
We also need to remove or prevent duplicates. 
In the degenerate case, `src/**/*.js src/**/*.js` should not beautify files twice.

Reply:
You are right! Added a check for this.